src/network/core/game_info.h

```diff
@@ -25,14 +25,25 @@
  * Version: Bytes:  Description:
  *   all      1       the version of this packet's structure
  *
+ *   6+       1       type of storage for the NewGRFs below:
+ *                      0 = NewGRF ID and MD5 checksum.
+ *                          Used as for version 5 and below, and for later
+ *                          game updates to the Game Coordinator.
+ *                      1 = NewGRF ID, MD5 checksum and name.
+ *                          Used for direct requests and the first game
+ *                          update to Game Coordinator.
+ *
  *   5+       4       version number of the Game Script (-1 is case none is selected).
  *   5+       var     string with the name of the Game Script.
  *
- *   4+       1       number of GRFs attached (n)
- *   4+       n * 20  unique identifier for GRF files. Consists of:
- *                     - one 4 byte variable with the GRF ID
- *                     - 16 bytes (sent sequentially) for the MD5 checksum
- *                       of the GRF
+ *   4+       1       number of GRFs attached (n).
+ *   4+       n * var identifiers for GRF files. Consists of:
+ *                     - 4 byte variable with the GRF ID.
+ *                       For v4, v5, and v6+ in case of type 0 and/or type 1.
```

Comment:
`type ` reference here is a bit lost in the context. Maybe use `storage type 1` or something?

Reply:
Added a bigger explanation before it.